### DISCUSSION TITLE
Map grains with rotation axis

### DISCUSSION THREAD

**Discussion by hwhouse** [mtex-toolbox/mtex#573] [Ask Anything] [answered]
Dear community：

I calculate the grain rotation of each grain during tensile deformation, now i can obtain the Rs(roation axis in sample coordinate) and Rc(rotation axis in crystal coordinate) of each pixel of a grain after deformation, how can i draw the Rs and Rc map of the grain respectively?

Best regards
Wang Hao

**Comment by kilir** [comment, marked as answer]
Hi,
assuming that your axes in specimen coordinates are of type vector3d and those in crystal coordinates are of type Miller you can use a colorcoding such as:
```matlab
axes_in_specimen_coordinates = vector3d.rand(10) % some random data
cKS =HSVDirectionKey(specimenSymmetry('1'))
rgb = cKS.direction2color(axes_in_specimen_coordinates)

axes_in_specimen_coordinates = Miller.rand(10,crystalSymmetry('222')) % some random data
cKC = HSVDirectionKey(axes_in_specimen_coordinates.CS):
rgb = cKC.direction2color(axes_in_specimen_coordinates);
```
Have a look at the documentation, for example here: https://mtex-toolbox.github.io/EBSDGROD.html 

Cheers,
Rüdiger
